charts/kubearchive/values.yaml

```diff
@@ -69,9 +69,9 @@ database:
   # provide your own database, set 'enabled: false' and modify the secret after deploying
   enabled: true
 
-  name: "{{ .Release.Name }}-database"
+  name: "{{ .Release.Name }}-rw"
```

Comment:
why `rw`? I guess it stands for `readwrite` but I don't see how this name is better than database

Reply:
I see, it's explained in the [architecture section](https://cloudnative-pg.io/documentation/1.15/architecture/) of the cnpg docs